script/generate-linux.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env brew ruby
+directories = ["_data/formula-linux", "api/formula-linux", "formula-linux"]
+FileUtils.rm_rf directories
+FileUtils.mkdir_p directories
+
+CoreTap.instance.formula_names.each do |n|
+  f = Formulary.factory(n)
+  json_filename = "#{f.name}.json"
+  IO.write("_data/formula-linux/#{json_filename.gsub('+', '_')}", JSON.pretty_generate(f.to_hash))
+  FileUtils.cp "_api_formula.json.in", "api/formula-linux/#{json_filename}"
+
+  html = IO.read "_formula_linux.html.in"
+  html.gsub!("title: $TITLE", "title: \"#{f.name}\"")
+  IO.write("formula-linux/#{f.name}.html", html)
+end
```

Comment:
Have a look at the optimized & simplified revisions to the original scripts here: Homebrew/formulae.brew.sh#157